path-compression/path_compresssion.go

```diff
@@ -72,7 +74,39 @@ func CompressPathToFile(pathToCompress string, enforceMaxFileSizeLimit bool) (st
 	}
 
 	compressedFilePath := filepath.Join(tempDir, filepath.Base(pathToCompress)+compressionExtension)
-	if err = archiver.Archive(filepathsToUpload, compressedFilePath); err != nil {
+
+	// Create output file
+	outFile, err := os.Create(compressedFilePath)
+	if err != nil {
+		return "", 0, nil, stacktrace.Propagate(err, "Failed to create output file '%s'.", compressedFilePath)
```

Comment:
`s/Failed to/An error occurred/`  https://github.com/kurtosis-tech/style-guide/blob/main/guides/go-style-guide.md#use-stacktracepropagate-or-stacktracenewerror-from-the-githubcomkurtosis-techstacktrace-package-whenever-possible we usually prefer this

Reply:
Ah, apologies, I missed that style guide.
I've pushed a fix for this in  bf837cb